compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -86,6 +85,13 @@ def __init__(
         self.container_uri = container_uri
         self.container_cmd_options = container_cmd_options
 
+        # We can remove this code once we rename the max_workers attribute on
+        # the Parsl HTEX to max_workers_per_node, which is more explicit.
+        max_workers_per_node = kwargs.pop("max_workers_per_node", None)
+        max_workers = max_workers_per_node or kwargs.get("max_workers", float("inf"))
```

Comment:
Is there a reason max_workers has a default of infinity but max_workers_per_node is None?  If they are supposed to be interchangeable shouldn't they be the same?

Reply:
The final result is that if `max_workers_per_node` and `max_workers` are not defined by the user, `max_workers` is set to `float("inf")`, which is what we want. We will also delete this code entirely once we add support for `max_workers_per_node` in the Parsl HTEX.

Reply:
The final result is that if neither `max_workers_per_node` or `max_workers` are defined by the user, `max_workers` is set to `float("inf")`, which is what we want. We will also delete this code entirely once we add support for `max_workers_per_node` in the Parsl HTEX.